packages/mobile/src/send/SendAmount/SendAmountValue.tsx

```diff
@@ -41,9 +45,16 @@ function SendAmountValue({
   return (
     <>
       <View style={styles.container}>
-        <BorderlessButton style={styles.maxButtonContainer} onPress={onPressMax}>
-          <Text style={styles.maxButton}>{t('max')}</Text>
-        </BorderlessButton>
+        {isOutgoingPaymentRequest ? null : (
```

Comment:
just checking that we want to render null instead of the "max" button if this is true, as the "max" button always existed in the previous version?

Reply:
The max button is new in 1.22.0. We don't want to render the max button on outgoing payment request.